### PR TITLE
Fix javadoc reported errors when switching to jdk11

### PR DESCRIPTION
**Pull request by emaldona**
- Address: https://github.com/dogtagpki/jss/issues/65
- use of `<tt>/</tt>` tags causes failures with jdk11 tools
- see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tt
- not a problem with either jdk8 or jdk10

### CODE REVIEW

**Comment by cipherboy** [comment]
So reading the attached Mozilla article, they do use `<code>` in a few places. Could we perhaps trade our `<tt>` elements for `<code>` blocks?

**Comment by emaldona** [comment]
Sure, I'll use that. Thanks for looking that up.

**Comment by emaldona** [comment]
I should add that when one is dealing with javadoc changes that need to be work for jdk11 and prior jdk's, such as 8, one uses "sudo alternatives ... " to switch, besides java and javac, also do it switch javadocdir. It makes a difference as jdk11 generates additional files such as several package-frame.html and some javascript ones. Perhaps we should add some note regarding this to the instructions given for package maintainers and contributors who likely need to support multiple versions of openjdk.

**Comment by emaldona** [comment]
Noticed that the PR makes a reference to #65 but should be to #66 instead, and error in the commit comments. It can be confusing. @cipherboy do you know an easy way to fix that?

**Comment by cipherboy** [comment]
Sure that sounds like a good idea. I'll draft some docs and have you review
them after merging this one.

**Comment by emaldona** [comment]
Okay, trying git commit --amend

File: org/mozilla/jss/ssl/SSLSocket.java [hunk 1]
```diff
@@ -1596,7 +1596,7 @@ private static native boolean isFipsCipherSuiteNative(int ciphersuite)
      * Returns a list of cipher suites that are implemented by NSS.
      * Each element in the array will be one of the cipher suite constants
      * defined in this class (for example,
-     * <tt>TLS_RSA_WITH_AES_128_CBC_SHA</tt>).
+     * <code>TLS_RSA_WITH_AES_128_CBC_SHA</code>).
```
Comment (cipherboy) [inline org/mozilla/jss/ssl/SSLSocket.java]: I force pushed a change reverting the parenthesis to the original location.

Comment (emaldona) [reply org/mozilla/jss/ssl/SSLSocket.java]: The changes you made are fine, ACK's feel free to push. Thanks.

**Comment by emaldona** [comment]
In my comment at the top it shows Address: #65 but it should be Address: #66, I can't edit that part.   @cipherboy could you please fix it? Thanks.

**Comment by cipherboy** [comment]
I think #65 is correct. That's the issue you created that's associated with this change, correct? #66 is the Pull Request, #65 is an issue.

**Comment by emaldona** [comment]
You are right. It's the PR that has the 66 which confused me.

**Pull request merged**